### PR TITLE
Add Registration CTA to bottom of all CoinFi News articles

### PR DESCRIPTION
**Pull request by hanchang**
https://app.asana.com/0/902343675126966/921631352220749

`alignLeft` is a janky solution I just threw together; the default is to be center aligned, so `alignLeft` is an optional param specifically for Twitter to make the layout look reasonable.

This PR also includes Heap instrumentation.

screen shot 2018-11-28 at 11 56 23 am
screen shot 2018-11-28 at 12 58 54 pm

### CODE REVIEW

File: app/javascript/App/bundles/NewsfeedPage/TwitterBody.tsx [hunk 1]
```diff
@@ -23,23 +24,26 @@ export default class NewsBody extends React.Component<Props, {}> {
...
-            {categories.map((category, index) => (
-              <div key={index} className="tag-alt">
-                {category.name}
-              </div>
-            ))}
-          </div>
-        )}
-        <Tweet tweetId={tweetId} />
-      </div>
+      <>
```
Comment (anthotsang) [inline app/javascript/App/bundles/NewsfeedPage/TwitterBody.tsx]: this doesn't look like it's necessary

File: app/javascript/App/bundles/NewsfeedPage/NewsBody.tsx [hunk 2]
```diff
@@ -165,6 +166,9 @@ export default class NewsBody extends React.Component<Props, State> {
             }}
           </RailsConsumer>
         </div>
+
+        <br />
+        <CallToAction />
```
Comment (anthotsang) [inline app/javascript/App/bundles/NewsfeedPage/NewsBody.tsx]: should this only be here if not logged in?

File: app/javascript/App/bundles/NewsfeedPage/TwitterBody.tsx (hunk 1 above)
Comment (hanchang) [reply app/javascript/App/bundles/NewsfeedPage/TwitterBody.tsx]: Great catch, thanks - removing...

File: app/javascript/App/bundles/NewsfeedPage/NewsBody.tsx (hunk 2 above)
Comment (hanchang) [reply app/javascript/App/bundles/NewsfeedPage/NewsBody.tsx]: Was hoping to not have to prop drill `loggedIn` but it's a better UX so will just bite the bullet, thanks.

**Pull request merged**